trestle/resources/config.ini

```diff
@@ -0,0 +1,33 @@
+[key-model-mapping]
+groups = trestle.oscal.catalog.Group
+controls = trestle.oscal.catalog.Control
+targets = trestle.oscal.target.Target
```

Comment:
Somewhere in this file we should have a preference for output (json vs yaml). I would like to default to json for now as it's less lossy.

Reply:
Created issue #64.